package.json

```diff
@@ -5,16 +5,16 @@
   "homepage": "https://earthfast.com",
   "license": "MIT",
   "scripts": {
-    "run": "export NODE_NO_WARNINGS=1 && npm run build && ./bin/run",
-    "dev": "export NODE_NO_WARNINGS=1 && npm run build && ./bin/dev",
+    "run": "export NODE_NO_WARNINGS=1 NODE_OPTIONS=--unhandled-rejections=none && npm run build && ./bin/run",
+    "dev": "export NODE_NO_WARNINGS=1 NODE_OPTIONS=--unhandled-rejections=none && npm run build && ./bin/dev",
     "debug": "DEBUG=1 npm run build && ./bin/dev",
     "build": "rm -rf ./dist && tsc",
     "clean": "rm -rf ./dist",
     "postpack": "rm -f oclif.manifest.json",
     "prepack": "npm run build && oclif manifest"
   },
   "bin": {
-    "earthfast": "./bin/run"
+    "earthfast": "export NODE_NO_WARNINGS=1 NODE_OPTIONS=--unhandled-rejections=none && ./bin/run"
```

Comment:
idk if i'm doing smth wrong but this is not working for me locally, and ai is saying this won't work the bin config can only contain a relative path to executable

Reply:
i'm able to run this locally 
```
ubuntu@ip-172-31-5-74:~/earthfast-cli$ npx earthfast
Command line utility for EarthFast

VERSION
  earthfast-cli/0.9.0 linux-x64 node-v20.18.1

USAGE
  $ earthfast [COMMAND]

TOPICS
  admin        Perform admin functions such as granting roles.
  bundle       Build EarthFast content bundles.
  key          Manage keys for signing transactions.
  node         Create and manage nodes.
  operator     Create and manage operators.
  project      Create and manage projects.
  reservation  Reserve content nodes for a project.

COMMANDS
  help    Display help for earthfast.
  status  Shows the current epoch status.
```

Reply:
on macos

Reply:
hmm i can run on macos as well
```
earthfast-cli|dm-keytar-deps-bugfix⚡ ⇒ npx earthfast
Command line utility for EarthFast

VERSION
  earthfast-cli/0.9.0 darwin-arm64 node-v20.10.0

USAGE
  $ earthfast [COMMAND]

TOPICS
  admin        Perform admin functions such as granting roles.
  bundle       Build EarthFast content bundles.
  key          Manage keys for signing transactions.
  node         Create and manage nodes.
  operator     Create and manage operators.
  project      Create and manage projects.
  reservation  Reserve content nodes for a project.

COMMANDS
  help    Display help for earthfast.
  status  Shows the current epoch status.
```